src/main/java/uk/gov/ons/ctp/common/time/DateTimeUtil.java

```diff
@@ -76,8 +78,7 @@ public static XMLGregorianCalendar stringToXMLGregorianCalendar(String string, S
       gregorianCalendar.setTime(date);
       result = DatatypeFactory.newInstance().newXMLGregorianCalendar(gregorianCalendar);
     } catch (ParseException e) {
-      log.error(String.format("%s - %s", e.getCause(), e.getMessage()));
-      log.error("Stack trace: " + e);
+      log.error(e, "Failed to parse date");
```

Comment:
order isn't what I'm used to seeing

Reply:
I'll sort it. I don't know why I did it that way